tests/e2e/ctl_v3_snapshot_test.go

```diff
@@ -380,12 +383,16 @@ func TestRestoreCompactionRevBump(t *testing.T) {
 
 	t.Log("(Re)starting the etcd member using the restored snapshot...")
 	epc.Procs[0].Config().DataDirPath = newDataDir
+
 	for i := range epc.Procs[0].Config().Args {
 		if epc.Procs[0].Config().Args[i] == "--data-dir" {
 			epc.Procs[0].Config().Args[i+1] = newDataDir
 		}
 	}
 
+	// Verify that initial snapshot is created by the restore operation
+	verifySnapshotMembers(t, epc, membersBefore)
```

Comment:
The cluster in this test case has only one member,  please consider to add a case verify that the members read from the snapshot are also expected in multiple members cluster in a separate PR.